_15thnight/forms.py

```diff
@@ -26,31 +36,52 @@ class RegisterForm(Form):
     )
     confirm = PasswordField(
         'Confirm Password',
-        validators=[DataRequired(), EqualTo('password', message='Passwords muct match.')]
-        )
+        validators=[
+            DataRequired(),
+            EqualTo('password', message='Passwords muct match.')
+        ])
     role = SelectField('User Role', choices=USER_ROLES)
-    clothes = BooleanField('Clothes')
-    shelter = BooleanField('Shelter')
-    food = BooleanField('Food')
-    other = BooleanField('Other')
+    categories = SelectMultipleField(
+        "Categories",
+        choices=[],
+        coerce=int
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(RegisterForm, self).__init__(*args, **kwargs)
+        self.categories.choices = [
+            (category.id, category.name) for category in Category.all()]
```

Comment:
This could fit on one line as:

`self.categories.choices = [(c.id, c.name) for c in Category.all()]`